packages/core-api/src/versions/2/delegates/methods.ts

```diff
@@ -82,11 +93,17 @@ const voterBalances = async request => {
 };
 
 export function registerMethods(server) {
+    const blockTime = config.getMilestone().blocktime;
```

Comment:
```suggestion
    const { activeDelegates, blockTime } = config.getMilestone();
```

:trollface:

Reply:
i'm not falling for your traps :danstop: